modules/quickstart-public-cloud/pages/setup.adoc

```diff
@@ -1,4 +1,24 @@
 [[quickstart-publiccloud-setup]]
 = Setting Up
 
-Completely leverage other's superior bandwidth via multimedia based interfaces. Enthusiastically disseminate superior processes with B2B web services. Dynamically brand leveraged leadership and top-line sources. Interactively.
\ No newline at end of file
+
+Public clouds provide {productname} under a Bring Your Own Subscription (BYOS) model.
+This means that you must register instances with the {scc}.
+For more information about registering {productname} with {scc}, see xref:installation:general-requirements.adoc[].
+
+Depending on the public cloud framework you are using, you can locate the {productname} images by searching for the  keywords [package]``suse``, [package]``manager``, [package]``proxy``, or [package]``BYOS``.
```

Comment:
These are not \[package\]s :stuck_out_tongue_winking_eye:

Reply:
No, they're not. We'll need to remove the semantic markup from this and the original.